### PR TITLE
updated doc on mesh coloring

### PR DESCRIPTION
**Pull request by mleotta**

### CODE REVIEW

**Comment by mleotta** [comment]
@danlipsa I updated the doc relative to the new mesh coloring options.  Can you review this?  Feel free to make suggested changed.

See it rendered in HTML here:
https://telesculptor--526.org.readthedocs.build/en/526/processingsteps.html#colorize-mesh

File: doc/processingsteps.rst [hunk 1]
```diff
@@ -363,6 +383,25 @@ The *Normals* option colors the mesh by surface normal direction, and the *NbPro
    A fused mesh colored by the number of views
    that see each part of the surface.
 
+Additional check boxes are available to enable removal of color
+on some surfaces that should not be colored.
+The *Remove masked surface color* option uses a mask video loaded
+into TeleSculptor and does not apply color at points under this mask.
```
Comment (danlipsa) [inline doc/processingsteps.rst:389]: This command can be used to hide the rough border of the generated mesh.

File: doc/processingsteps.rst [hunk 2]
```diff
@@ -363,6 +383,25 @@ The *Normals* option colors the mesh by surface normal direction, and the *NbPro
...
+The *Remove occluded surface color* option enables visibility checking
+such that occluded surfaces will not receive color.
+The *Occlusion threshold* varies the sensitivity of
+the occlusion depth test.
+
```
Comment (danlipsa) [inline doc/processingsteps.rst:394]: Negative values cause larger occluded regions while positive values cause smaller occluded regions.

**Comment by danlipsa** [comment]
Looks good. I added a few clarifications. Feel free to change them or ignore them all together.

File: doc/processingsteps.rst [hunk 3]
```diff
@@ -363,6 +383,25 @@ The *Normals* option colors the mesh by surface normal direction, and the *NbPro
...
    that see each part of the surface.
 
+Additional check boxes are available to enable removal of color
+on some surfaces that should not be colored.
+The *Remove masked surface color* option uses a mask video loaded
+into TeleSculptor and does not apply color at points under this mask.
+The *Remove occluded surface color* option enables visibility checking
+such that occluded surfaces will not receive color.
+The *Occlusion threshold* varies the sensitivity of
+the occlusion depth test.
```
Comment (mleotta) [inline doc/processingsteps.rst:393]:
```suggestion
the occlusion depth test.
Negative values cause larger occluded regions while
positive values cause smaller occluded regions.
```

File: doc/processingsteps.rst (hunk 2 above)
Comment (mleotta) [reply doc/processingsteps.rst:396]: done

File: doc/processingsteps.rst (hunk 1 above)
Comment (mleotta) [reply doc/processingsteps.rst:389]: I think this statement is specific to the AP Hill sample data which has darker or black pixel at the image border.

**Pull request merged**